usecases/memwatch/monitor.go

```diff
@@ -311,6 +312,10 @@ func EstimateObjectMemory(object *models.Object) int64 {
 	return int64(len(object.Vector)*4 + 30)
 }
 
+func EstimateStorObjectMemory(object *storobj.Object) int64 {
```

Comment:
@dirkkul to estimate the size of an object more accurately, do you think the right place for this would be for [Object.parseObject](https://github.com/weaviate/weaviate/blob/e56f6ef44b7f29ec133a9e10362ef6638c3eef5f/entities/storobj/storage_object.go#L990) to add this size of the object to `models.Object`?

Reply:
In general, this does not need to be super accurate. We want to avoid OOM situations and this mechanism is not designed to use every last byte so some fuzzyness is ok.

Where is the 49 below coming from? Would be great to add an explanation

Reply:
Actually it is not 49, but 46. This value is derived from summing the memory occupancy of some data fields of `storobj.Object` such as `uint8`, `uint64`, etc.

I add more information about it in the comments of the function.